src/Quizzer.API.DAL/Entities/Answer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AutoMapper;
+
+namespace Quizzer.API.DAL.Entities {
+    public record Answer : EntityBase {
+        public required string Text { get; set; }
+        public string? PictureUrl { get; set; }
+        public required bool IsCorrect { get; set; }
+        public int? Order { get; set; }
+
+        public Guid QusetionId { get; set; }
+        public ICollection<SelectedAnswer> SelectedAnswers { get; set; } = new List<SelectedAnswer>();
+    }
+
+    public class AnswerMapperProfile : Profile {
+        public AnswerMapperProfile() => CreateMap<Answer, Answer>();
+    }
```

Comment:
Well i guess that this should not be probelm, because in CB it is in one file too. But if you wnat it, it is not a problem. So, should i do it?
![image](https://github.com/vecerap/iw5-2023/assets/80791135/002af0be-857b-47ac-b41c-467e7af025de)

Reply:
If this approach is used in CB, I would leave it as you have it now.